dgraph/cmd/zero/oracle.go

```diff
@@ -55,21 +58,37 @@ type Oracle struct {
 	syncMarks   []syncMark
 }
 
+func initKeyCommit() *z.Tree {
+	f, err := ioutil.TempFile("", "dgraph-keycommit")
+	x.Check(err)
+	mf, err := z.OpenMmapFileUsing(f, 1<<30, true)
+	if err != z.NewFile {
```

Comment:
Should we use the `/tmp` directory for now? One problem with this is that users may delete the `/tmp` directory and that would cause `dgraph` to crash.

Reply:
Keeping it as it is for now. We can add `--tmp` directory for `zero` as well if needed.